core/unit_test/TestTeam.hpp

```diff
@@ -1616,6 +1616,73 @@ struct TestTeamPolicyHandleByValue {
 
 }  // namespace
 
+namespace {
+template <typename ExecutionSpace>
+struct TestRepeatedTeamReduce {
+  static constexpr int ncol = 1500;  // nothing special, just some work
+
+  KOKKOS_FUNCTION void operator()(
+      const typename Kokkos::TeamPolicy<ExecutionSpace>::member_type &team)
+      const {
+    // non-divisible by power of two to make triggering problems easier
+    constexpr int nlev = 129;
+    constexpr auto pi  = Kokkos::numbers::pi;
```

Comment:
```suggestion
    constexpr auto pi  = Kokkos::Experimental::numbers::pi;
```
Since we did the promotion after the 3.7 release?